<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="BINCDV-I10165" name="Regression.INT.INT_Price_Req_Currency_Depnd_fields_update_Locked" tests="1" failures="1" timestamp="10 Jul 2017 09:45:54 GMT" time="178.100" errors="0">
  <testcase name="INT_Price_Req_Currency_Depnd_fields_update_Locked_Validation" time="178.100" classname="Regression.INT.INT_Price_Req_Currency_Depnd_fields_update_Locked">
    <failure type="java.lang.AssertionError" message="Test Case failed. ErrorMsg: PriceLineItem from related list not selected succesfully">
      <![CDATA[java.lang.AssertionError: Test Case failed. ErrorMsg: PriceLineItem from related list not selected succesfully
	at org.testng.Assert.fail(Assert.java:94)
	at com.cognizant.framework.Report.updateTestLog(Report.java:171)
	at com.framework.testbase.TestBase.chooseItemFromLeftTable(TestBase.java:3035)
	at Regression.INT.INT_Price_Req_Currency_Depnd_fields_update_Locked.INT_Price_Req_Currency_Depnd_fields_update_Locked_Validation(INT_Price_Req_Currency_Depnd_fields_update_Locked.java:94)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
    </failure>
  </testcase> <!-- INT_Price_Req_Currency_Depnd_fields_update_Locked_Validation -->
</testsuite> <!-- Regression.INT.INT_Price_Req_Currency_Depnd_fields_update_Locked -->
